### DISCUSSION TITLE
wsl2 ubuntu

### DISCUSSION THREAD

**Discussion by HUAHUAI23** [iamcco/markdown-preview.nvim#499] [Q&A] [answered]
after entering :MarkdownPreview command，there is no expected response and no browser window open，
my neovim version is NVIM v0.7.2，
i have installed xdg-utils via sudo apt-get install -y xdg-utils and set a alias edge='/mnt/c/"Program Files (x86)"/Microsoft/Edge/Application/msedge.exe'，and also let g:mkdp_open_to_the_world = 1
![image](https://user-images.githubusercontent.com/43649186/188317254-fa4e389c-a637-4156-b6dd-27a2a6d7cdd3.png)

**Comment by VvSsNPk** [comment, marked as answer]
is the isuue solved for me same problem
